lib/AuthorizerModule.js

```diff
@@ -48,9 +50,19 @@ class AuthorizerModule {
                 (failedPrincipalId) => {
                     logger.info('There was an error fulfilling the promise');
                     return failedPrincipalId;
-            });
+                });
+
+        var facebookPromise = this.facebookMod.callIdProvider(event.authorizationToken)
+            .then((principalId) => {
+                return principalId;
+            })
+            .catch(
+                (failedPrincipalId) => {
+                    logger.info('There was an error fulfilling the promise');
+                    return failedPrincipalId;
+                });
 
-        return Promise.all([googlePromise]).then(
+        return Promise.all([googlePromise, facebookPromise]).then(
```

Comment:
There is an existing test that validates a successful Google call. Can you make one similar for the FB module success?